### DISCUSSION TITLE
Events Page Creation

### DISCUSSION THREAD

**Discussion by cihatkapusuz** [bounswe/bounswe2020group6#351] [Ideas]
We will discuss about the features that we will provide at events page.

**Comment by emilcan-arican** [comment]
Users should be able to state whether event will be online or physical. Since we are experiencing a pandemic, it is an important feature.

**Comment by DogukanKalkan** [comment]
There are two options for creation of events. We can either use an API to list all the recent events, conferences or the users will create on their own.

**Reply by Tubagc** [reply]
If events are created inside the app, there should be some fields named,
- title
- description
- location
- online/offline
- tags
- date

**Reply by Tubagc** [reply]
- abstract submission deadline
- project submission deadline
